synology-packages.md

```diff
@@ -51,26 +50,35 @@ Bubblewrap allows us to run programs in a basic container so that we can use new
 
 ## Bubblewrap Manual Step - Recommended Option
 
-1. Create a scheduled task within DSM:
+1. Create a triggered task within DSM:
 	- User: `root`
   	- Event: `Boot-up`
-1. For the `Run Command` enter:
+2. For the `Run Command` enter:
```

Comment:
1. 

Wiki auto increments

Reply:
Noted. Didn't knew about this, new to this.